tox.ini

```diff
@@ -1,28 +1,86 @@
-# Copyright 2024 Canonical Ltd.
+# Copyright 2025 Ubuntu
 # See LICENSE file for licensing details.
+
 [tox]
-requires =
-    tox>=4
-envlist = make-prs
+no_package = True
+skip_missing_interpreters = True
+env_list = format, lint, static, unit
+min_version = 4.0.0
+
+[vars]
+src_path = {tox_root}/src
+tests_path = {tox_root}/tests
+;lib_path = {tox_root}/lib/charms/operator_name_with_underscores
+all_path = {[vars]src_path} {[vars]tests_path}
 
 [testenv]
-basepython = python3
-setenv =
-  PY_COLORS=1
-passenv =
-  GITHUB_TOKEN
-
-[testenv:make-prs]
```

Comment:
Maybe it's less needed when the tutorial is smaller, etc, but couldn't we keep this for now?